met-web/tests/unit/components/setEnvVars.tsx

```diff
@@ -0,0 +1,6 @@
+export function setupEnv() {
+    process.env.REACT_APP_KEYCLOAK_CLIENT = 'met-web';
```

Comment:
I think we may need to tweak this in the future

Reply:
any suggestions?